common/core/src/test/resources/simple.json

```diff
@@ -40,6 +40,7 @@
       }
     ],
     "allow_repeated":true,
-    "completed_label": "Completed"
+    "completed_label": "Completed",
+    "mutually_exclusive_set_id": "HCA_Approve_ServiceNow_1"
```

Comment:
This test case isn't very meaningful (this property doesn't make sense if there is only 1 action with it).

Instead of this, go ahead and fix Concur connector to use this (and get rid of the Open button like Ashley recommended).